components/mime.rst

```diff
@@ -0,0 +1,88 @@
+.. index::
+    single: MIME
+    single: Components; MIME
+
+The MIME Component
+==================
+
+    The MIME component provides utilities related to MIME types, such as
+    guessing the MIME type of a given file.
```

Comment:
If the scope of the component is just media types rather than MIME as a whole, could it be named as such instead?